packages/autorest.typescript/test/integration/generated/datasearch/package.json

```diff
@@ -85,7 +87,7 @@
     "unit-test:node": "cross-env TEST_MODE=playback npm run integration-test:node",
     "unit-test:browser": "echo skipped",
     "integration-test": "npm run integration-test:node && npm run integration-test:browser",
-    "integration-test:node": "mocha -r esm --require ts-node/register --timeout 1200000 --full-trace test/*.ts --reporter ../../../common/tools/mocha-multi-reporter.js",
+    "integration-test:node": "cross-env TS_NODE_COMPILER_OPTIONS=\"{\\\"module\\\":\\\"commonjs\\\"}\" mocha -r esm --require ts-node/register --timeout 1200000 --full-trace --reporter ../../../common/tools/mocha-multi-reporter.js test/*.ts",
```

Comment:
Could you help me understand why we need below config? My understanding is it is commonjs module by default if without type:module in package.json.

```
cross-env TS_NODE_COMPILER_OPTIONS=\"{\\\"module\\\":\\\"commonjs\\\"}\
```

Reply:
> dev-tool will not do any workaround to mitigate import errors if no js suffix. Is it correct or not?

That's right, ".js" file are consider in cjs format.  But our target in tsconfig is ES2017, which is ESM, so we override it to commonjs when using ts-node.  I don't remember about the exact mocha v10 change but without env variable it gives errors.